vision-zero

```diff
@@ -24,9 +24,6 @@ TABLES_TO_IGNORE = [
     "public.change_log_people_cris",
     "public.change_log_people",
     "public.change_log_units",
-    "public.change_log_people_edits",
-    "public.change_log_units_edits",
-    "public.change_log_crashes_edits",
```

Comment:
saying bye to these, also should we delete all the edit layer tables from the `./metadata/databases/default/tables`? i just noticed those dont get automatically deleted when you drop tables

Reply:
cool—yeah, i noticed we have a _ton_ of crufty metadata in this directory. i'll open a separate issue to do a fresh `hasura metadata export` once we get all of our current PRs merged. we have quite a lot of metadata changes floating out there right now 🙏

Reply:
https://github.com/cityofaustin/atd-data-tech/issues/19577